.coveragerc

```diff
@@ -4,6 +4,7 @@ source = gocept.cache
 
 [report]
 precision = 2
+fail_under = 98
```

Comment:
It would be nice to mention this line in a ticket for the Python 2 support dropping so it can be set back to 100 once that ticket is resolved.

Reply:
Good idea. I opened #6 to keep track of it.